exposed-jodatime/src/test/kotlin/org/jetbrains/exposed/JodaTimeTests.kt

```diff
@@ -239,7 +335,7 @@ fun equalDateTime(d1: DateTime?, d2: DateTime?) = try {
     false
 }
 
-val today: DateTime = DateTime.now().withTimeAtStartOfDay()
+val today: DateTime = DateTime.now(DateTimeZone.UTC).withTimeAtStartOfDay()
```

Comment:
Several tests in `exposed-jodatime` module were failing because they were expecting `today` to be in UTC time zone (as set in the init block), but were getting it in Tokyo time zone. I tried to set the time zone to UTC both before and after each test like this, but it didn't fix the failing tests.
```
@Before
@After
fun setTimeZone() {
    DateTimeZone.setDefault(DateTimeZone.UTC)
}
```
Specifying the time zone fixed it.